community/concepts/deep-then-wide.md

```diff
@@ -0,0 +1,30 @@
+# deep then wide
```

Comment:
(as per review comment in the glossary) -- i'm nto sure deep-then-wide needs a concept here.

Reply:
It was added on the discourse by @Beanow so I thought it might be worth explaining. I can definitely see that someone might not get what is meant, so that alone would justify adding it (for me).